localstack/utils/common.py

```diff
@@ -607,8 +611,13 @@ def download(url, path, verify_ssl=True):
                     LOG.debug('Empty chunk %s (total %s) from %s' % (chunk, total, url))
             f.flush()
             os.fsync(f)
+        if os.path.getsize(path) == 0:
+            LOG.error('Zero bytes downloaded from %s, retrying' % url)
```

Comment:
Can we please reduce the log level to `LOG.info` or `LOG.warning`? We're generally only using `error` for fatal errors.